scripts/mkcloud

```diff
@@ -1011,25 +1063,37 @@ Steps:
     cleanup:        kill all running VMs, zero out boot sectors of all LVM volumes
     prepare:        create LVM volumes, setup libvirt networks
     setupadmin:     create the admin node and install the cloud product
-    prepareinstcrowbar: add repos and install crowbar packages
+    prepareinstcrowbar:
+                    add repos and install crowbar packages
```

Comment:
many unrelated changes in this commit - even unrelated to the PR
please split it off into separate PR